src/gtk.lisp

```diff
@@ -45,25 +47,32 @@
 (s:defsketch tile-test ((s:title "tile")
                         (choose-xy nil))
   (s:background s:+black+)
-  (s+:with-fit ((* +tiles-count-h+ +tile-side+)
-                (* +tiles-count-v+ +tile-side+)
-                s:width s:height)
```

Comment:
Maybe return with-fit back? It just allows to properly handle window resizing by using `sketch:translate` and `sketch:scale` to "fake" needed window size.

You probably want something like that here:
```lisp
(s+:with-fit (900 800 s:width s:height)
  ;; first two args are width and height you ask for
  ;; s:width and s:height are true values for current width and height
  ...
```